src/main/resources/schemas/nhinc/common/AuditQueryLog.xsd

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsd:schema xmlns:xsd="http://www.w3.org/2001/XMLSchema"
+            targetNamespace="urn:gov:hhs:fha:nhinc:common:auditquerylog"
+            xmlns:tns="urn:gov:hhs:fha:nhinc:common:auditquerylog"
+            xmlns:adtmsg="http://nhinc.services.com/schema/auditmessage">
+    <xsd:import schemaLocation="../../../schemas/ihe/auditmessage.xsd" namespace="http://nhinc.services.com/schema/auditmessage"/>
+    <xsd:complexType name="QueryAuditEventsRequestType">
+        <xsd:sequence>
+            <xsd:element name="UserId" minOccurs="0" type="xsd:string" />
+            <xsd:element name="EventType" minOccurs="0" type="xsd:string" />
+            <xsd:element name="EventOutcomeIndicator" minOccurs="0" type="xsd:integer" />
+            <xsd:element name="eventBeginDateTime" type="xsd:dateTime" />
+            <xsd:element name="eventEndDateTime" minOccurs="0" type="xsd:dateTime" />
+            <xsd:element name="RemoteHcid" minOccurs="0" type="xsd:string" />
+        </xsd:sequence>
+    </xsd:complexType>
+    <xsd:element name="QueryAuditEventsRequestType" type="tns:QueryAuditEventsRequestType"/>
+    <xsd:complexType name="QueryAuditEventsResponseType" >
+        <xsd:sequence>
+            <xsd:element name="UserId" minOccurs="0" type="xsd:string" />
+            <xsd:element name="EventType"  type="xsd:string" />
+            <xsd:element name="EventId"  type="xsd:string" />
+            <xsd:element name="EventOutcomeIndicator"  type="xsd:integer" />
+            <xsd:element name="EventTimestamp" type="xsd:dateTime" />
+            <xsd:element name="RemoteHcid" minOccurs="0" type="xsd:string" />
+            <xsd:element name="RelatesTo" minOccurs="0" type="xsd:string" />
+            <xsd:element name="Direction"  type="xsd:string" />
+            <xsd:element name="Id"  type="xsd:long" />
+            <xsd:element name="RequestMessageId"  minOccurs="0" type="xsd:string" />
+        </xsd:sequence>
+    </xsd:complexType>
+    <xsd:element name="QueryAuditEventsResponseType" type="tns:QueryAuditEventsResponseType"/>
+    <xsd:complexType name="QueryAuditEventsRequestByRequestMessageId">
+        <xsd:sequence>
+            <xsd:element name="RequestMessageId"  minOccurs="0" type="xsd:string" />
+        </xsd:sequence>
+    </xsd:complexType>
+    <xsd:element name="QueryAuditEventsRequestByRequestMessageId" type="tns:QueryAuditEventsRequestByRequestMessageId"/>
+    
+    <xsd:complexType name="QueryAuditEventsResponseByRequestMessageId" >
+        <xsd:sequence>
+            <xsd:element name="UserId" minOccurs="0" type="xsd:string" />
+            <xsd:element name="EventType"  type="xsd:string" />
```

Comment:
Can we use QueryAuditEventsResponseType instead of defining a new QueryAuditEventsResponseByRequestMessageId. They both have the same elements.